R/tm_g_distribution.R

```diff
@@ -542,9 +542,14 @@ srv_distribution <- function(id,
       datasets = data
     )
 
+    qenv <- teal.code::eval_code(
+      data(),
+      'library("ggplot2");library("forcats");library("dplyr")'
```

Comment:
There is no `forcats` but there is `rlistings` in the default example from the module

<img width="806" alt="image" src="https://github.com/user-attachments/assets/af64bfd9-2acf-417e-aa60-cdf9a7d03f4c" />


```suggestion
      'library("ggplot2");library("rlistings");library("dplyr")'
```

`forcats` is added only under 2 circumnstanes under those 2 `if` statements.
Maybe you can extend the `qenv` there with a `library(forcats)` call

<img width="520" alt="image" src="https://github.com/user-attachments/assets/55ffe4d6-7a8d-4888-b713-d687d4aebb6b" />

Reply:
forcats added on these two conditions, also added generics in once other code branch.